user_manual/talk/advanced_features.rst

```diff
@@ -145,8 +145,11 @@ After stopping a recording, the server will take some time to prepare and save t
 Recording consent
 ^^^^^^^^^^^^^^^^^
 
-For compliance reasons with various privacy rights, it is possible to ask participants for consent to be recorded before joining the call.
-The system administration can enable mandatory consent for all conversations system-wide or allow moderators to configure this option on a conversation level. If your system is set up accordingly, moderators can access the following option in the conversation settings:
+For compliance reasons with various privacy rights, it is possible to ask participants for consent to be recorded before joining the call. The system administration has the flexibility to utilize this feature in several ways:
```

Comment:
Maybe also add a screenshot form the administration settings :) ?

Reply:
It is a user manual, so no